### PR TITLE
Live indicator update

### PR DESCRIPTION
**Pull request by uplne**
The https://github.com/guardian/frontend/pull/7637 made the position of the live indicator a little bit off:
screen shot 2015-01-08 at 15 50 36

This should be fixing it and removing some spans in the process:
screen shot 2015-01-08 at 15 50 03

### CODE REVIEW

File: static/src/stylesheets/module/facia/_slices.scss [hunk 1]
```diff
@@ -171,8 +171,8 @@ Hence why a greater depth of selector specificity is needed.
         display: none;
     }
 
-    .fc-item__live-indicator .live-pulse-icon {
-        display: none;
+    .live-pulse-icon:before {
+        content: none;
```
Comment (ironsidevsquincy) [inline static/src/stylesheets/module/facia/_slices.scss]: just a question, what's you're opinion on keeping all styles for a selector in the same place? e.g, moving this into `_live-blog.scss`, inside `.live-pulse-icon:before`?

```
...
    @include animation(live-pulse 1s infinite);

    .fc-slice--popular & {
        content: none;
    }
}
```

File: static/src/stylesheets/module/content/_live-blog.scss [hunk 2]
```diff
@@ -105,6 +105,10 @@ $block-padding-right: $gs-gutter;
 
 .blog__live {
     font-weight: bold;
+
+    &.live-pulse-icon:before {
+        margin-bottom: -1px;
+    }
 }
 
 .live-pulse-icon:before {
```
Comment (ironsidevsquincy) [inline static/src/stylesheets/module/content/_live-blog.scss]: should this live in a more global file, as it's used on fronts and live blogs?

File: static/src/stylesheets/module/facia/_slices.scss (hunk 1 above)
Comment (uplne) [inline static/src/stylesheets/module/facia/_slices.scss]: Yes I think it's a good thing to do. I want this to deploy as a quick fix but I will move it to one file in another PR.

**Comment by phamann** [comment]
:+1:

**Pull request merged**